configuration/concepts/adult_hiv_initial_form.csv

```diff
@@ -0,0 +1,97 @@
+f7bfba0e-35b2-41dc-8890-d36ecdfc7a22,,a2065636-5326-40f5-aed6-0cc2cca81ccc;b497171e-0410-4d8d-bbd4-7e1a8f8b504e,Tabac,Tabacco,,,Misc,Coded,,
+80aa32e3-64b2-4014-ab4d-ef475615af05,,,Orale,Orally,,,Misc,N/A,,
+e273f66e-f349-47e7-af9f-6c80195136bc,,,Injectable inhalation,Injectable inhalation,,,Misc,N/A,,
+a54aab80-7d10-43af-a1d9-0a81a88fbe8b,,80aa32e3-64b2-4014-ab4d-ef475615af05;e273f66e-f349-47e7-af9f-6c80195136bc,Type de drogue (partie é compléter par les spécialistes),Type of drug use (to be completed by specialist),,,Misc,Coded,,
+9a7ad1b1-942f-43b4-b280-5df81b515135,c6faef09-0a6c-4f3b-98c6-a47c7299cf5b;619a9177-1625-44d6-83e5-21501d3d272a;df3f93c9-b3ec-4756-b5b8-00406428dc32;00fd3693-409e-45ad-bb89-d1058fff4b7a;28e491d0-e1fc-4e4a-9ad5-4edf634ff341;16c12c6b-f734-4fd6-974e-8088f5b1e013;c1a0d403-2597-469c-a8df-8b424630f3ae;9d9a399a-d272-4d39-a5b3-e8f674aed8cf;6aa07a98-1ace-4cd5-bad6-7e04507bc0ab;56516172-9d35-4e52-9d98-00ff82235aa6;83624a21-972b-40e2-af3e-1a42612f4756;a32c47d6-ff4b-4f0b-b8f6-dea611f17f59;94c4e73a-3a3a-443f-8ee5-88fa2fda4c4d;9fc81579-7daa-4064-91d7-2d49315f3837;6dfb72f4-d3e0-4605-a8c5-3e3a672d7cef;f7bfba0e-35b2-41dc-8890-d36ecdfc7a22;a54aab80-7d10-43af-a1d9-0a81a88fbe8b,,III.2 Médicaux,III.2 Medical,,,Misc,N/A,,
+fd43f382-0f14-4740-ad8f-3ec38b05fd14,,a2065636-5326-40f5-aed6-0cc2cca81ccc;b497171e-0410-4d8d-bbd4-7e1a8f8b504e,ATCD prescription ARV,ATCD prescription ARV,,,Misc,Coded,,
+02f7b1ff-1bfc-46b8-811d-bf524db681cb,,,Si traitement ARV antérieur; Date début,If previous ARV treatment; start date,,,Misc,Date,,
+fef74daa-2a47-4c4e-b9c2-d18078d8e3be,,,Date de fin (le cas échéant),End Date  ,,,Misc,Date,,
+81ff1578-da33-44f3-bdbe-71d7706ecd87,,,Formation sanitaire,Health Facility,,,Misc,Text,,
+09704fda-7fe3-45bf-a993-9b5eb7b69ed9,,,Transfert avec dossier,Transfer with file,,,Misc,N/A,,
+c74e2fd5-1005-49e8-bf2a-64aabd2c662a,,,Déjé été sou ARV,Already been ARV,,,Misc,N/A,,
+51d7342d-d8ad-4061-97f3-c32724a47694,,,PTME,PMTCT,,,Misc,N/A,,
+aeecf2e8-4746-4023-bc39-efdd83ce4e94,,09704fda-7fe3-45bf-a993-9b5eb7b69ed9;c74e2fd5-1005-49e8-bf2a-64aabd2c662a;51d7342d-d8ad-4061-97f3-c32724a47694,Circonstances du traitement ARV antérieur,Circumstances of previous ARV treatment,,,Misc,Coded,,
+3b2a2ded-60f6-4818-a4f4-10f296f53ae1,,,Médicament ARV prescrits,Prescribed ARV drug,,,Misc,Text,,
+29538d59-7239-475d-b95c-3e493f8b88fe,,a2065636-5326-40f5-aed6-0cc2cca81ccc;b497171e-0410-4d8d-bbd4-7e1a8f8b504e,Traitement toujours en cours,Treatment still in progress,,,Misc,Coded,,
+db3088d9-cc5c-433f-8e47-5e29881dd9c5,,a2065636-5326-40f5-aed6-0cc2cca81ccc;b497171e-0410-4d8d-bbd4-7e1a8f8b504e;Do Not Know,Cotrimoxazole (Prophylaxie en cours),Cotrimoxazole (Prophylaxsis in progress),,,Misc,Coded,,

```

Comment:
@lauravignoli  Could you please double check the above concept? Noticed the following issues:
`Fully specified name:fr` =`Do Not Know` : the text is in English
`Fully specified name:en` = `Cotrimoxazole (Prophylaxie en cours)`: the text is in French and seems unrelated to its fr counterpart
`Short name:fr` =`Cotrimoxazole (Prophylaxsis in progress)`: the text is english
`Short name:en` is empty

Reply:
I decided to use some concepts with the name instead of uuid after I had to update the uuid because it was a duplicate. When there's no duplicates (which should be always) uuid or concept name can be used in-distinctively.

Remember that in this file, for the name, the first column is fr and the second en. I removed the short names when they are the same as the full name.

Reply:
Thanks for the clarification @lauravignoli, I missed that  'Do Not Know' was part of the answer